pipe-cli/src/api/metadata.py

```diff
@@ -39,22 +39,44 @@ def find(cls, identifier, entity_class):
 
     @classmethod
     def load(cls, entity_id, entity_class):
+        return cls.load_all_for_ids([entity_id], entity_class)[0]
+
+    @classmethod
+    def load_all_for_ids(cls, entity_ids, entity_class):
         api = cls.instance()
-        data = json.dumps([{
-            'entityId': entity_id,
-            'entityClass': str(entity_class).upper()
-        }])
+        data = json.dumps([cls.convert_to_entity_vo(entity_id, entity_class) for entity_id in entity_ids])
         response_data = api.call('metadata/load', data=data, http_method='POST')
         if 'payload' in response_data:
-            for response_data_item in response_data['payload']:
-                return MetadataModel.load(response_data_item)
+            return [MetadataModel.load(response_data_item) for response_data_item in response_data['payload']]
         if 'status' in response_data and response_data['status'] == "OK":
-            return MetadataModel()
+            return [MetadataModel()]
         if 'message' in response_data:
             raise RuntimeError(response_data['message'])
         else:
             raise RuntimeError("Failed to load metadata.")
 
+    @classmethod
+    def load_metadata_mapping(cls, entity_ids, entity_class):
+        metadata_list = Metadata.load_all_for_ids(entity_ids, entity_class)
+        metadata_mapping = dict()
+        for metadata_entry in metadata_list:
+            metadata_data_dict = {}
+            for key, data in metadata_entry.data.iteritems():
```

Comment:
We need to use `from future.utils import iteritems` to make it compatible with Python 2 & 3

Reply:
Done